core/src/http/AuthHttpClient.js

```diff
@@ -576,6 +576,43 @@ export class AuthHttpClient {
         };
     }
 
+    /**
+     * get information about a particular standing order
+     *
+     * @param {*} accountId
```

Comment:
we should specify the type as much as we can because we don't use flow in this file